src/GitExtensions.GerritPlugin/GitExtensions.GerritPlugin.csproj.user

```diff
@@ -6,7 +6,7 @@
     <!-- 'latest' or 'v3.1' (= tag from GitHub releases) or 'v3.1.0.5877' (= build number from AppVeyor)-->
     <GitExtensionsReferenceVersion>latest</GitExtensionsReferenceVersion>
     <!-- 'GitHub' or 'AppVeyor' -->
-    <GitExtensionsReferenceSource>AppVeyor</GitExtensionsReferenceSource>
+    <GitExtensionsReferenceSource>GitHub</GitExtensionsReferenceSource>
```

Comment:
@mast-eu @maraf do you know what this property is used for?

Reply:
This one specifies a source where Git Extensions realease is downloaded from (from referencing assemblies and running debug instance).
GitHub is the default value. AppVeyor is used for "latest"/dev builds.

As the plugin should build against stable release, from GitHub, this property could be removed from the csproj.

Reply:
should I remove this line complete or is this change ok in this way?

Reply:
@mast-eu I'm not sure about your comment bellow and we should synchronize on this one. In my plugins, I'm always targeting a specific version of Git Extensions.
If I understand you correctly, you are saying that we should always develop against latest nightly build of Git Extensions and when merging to a release branch, change the reference to a specific version. Correct? But it could potentialy create issues, isn't it? Some APIs might be completly missing, or be different.

Reply:
@maraf, yes, you understood me correctly. 
In case of different APIs there would be additional changes needed in the release branch, i.e. reverting a commit that is not yet releasable, because the corresponding GE version is not yet released. This would than be included in a future release.

Obviously, my approach doesn't have to be the best one. It's just the best one I could come up with 🤷 
I'm open for improvements. Let's concentrate discussions in https://github.com/gitextensions/GitExtensions.GerritPlugin/issues/24.